client/my-sites/hosting/main.js

```diff
@@ -76,11 +76,10 @@ class Hosting extends Component {
 
 		const getUpgradeBanner = () => (
 			<UpsellNudge
```

Comment:
With the changes from the A/B test, I think we may want to add an early return to this function that returns the generic language when we're in the eCommerce trial, and let the experiment continue for all other plans. Flagged for the experimenting team here: pbxNRc-2f8-p2#comment-4030

Reply:
I agree with doing an early return if the user is on an eCommerce trial. I'm going to launch the experiment as soon as I'm back on Monday so I'd like to keep the experiment as designed, which includes linking directly to the checkout.

Once we know which variation works better then I think we can work to make it more generic.